app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -89,9 +96,8 @@ class StateFragmentPresenter @Inject constructor(
   private val explorationProgressController: ExplorationProgressController,
```

Comment:
ExplorationDataController is not in use. Please remove it.

Reply:
It's been removed.